modules/wordads/wordads.php

```diff
@@ -476,6 +492,11 @@ function get_ad_snippet( $section_id, $height, $width, $location = '', $css = ''
 		$data_tags = $this->params->cloudflare ? ' data-cfasync="false"' : '';
 		$css = esc_attr( $css );
 
+		$loc_id = 100;
+		if ( isset( self::$ad_location_ids[$location] ) ) {
+			$loc_id = self::$ad_location_ids[$location];
```

Comment:
```suggestion
			$loc_id = self::$ad_location_ids[ $location ];
```

Just to make our linters happy :)